docs/basic-dsa/Strings/Reversing a string.md

```diff
@@ -0,0 +1,98 @@
+---
+id: Reversing a string
```

Comment:
replace `id: Reversing a string` with `id: reversing-a-string` for best practices